eng/pipelines/linux.yml

```diff
@@ -123,15 +123,15 @@ stages:
           - redhatHelixQueue: RedHat.6.Amd64.Open
 
           - ${{ if eq(parameters.fullMatrix, 'false') }}:
-            - linuxDefaultQueues: Centos.7.Amd64.Open+RedHat.7.Amd64.Open+Debian.9.Amd64.Open+Ubuntu.1604.Amd64.Open+Ubuntu.1804.Amd64.Open+SLES.15.Amd64.Open+\(Fedora.29.Amd64.Open\)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-29-09ca40b-20190508143249
-            - alpineQueues: \(Alpine.39.Amd64.Open\)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.9-helix-09ca40b-20190620160617
+            - linuxDefaultQueues: Centos.7.Amd64.Open+RedHat.7.Amd64.Open+Debian.9.Amd64.Open+Ubuntu.1604.Amd64.Open+Ubuntu.1804.Amd64.Open+SLES.15.Amd64.Open+\(Fedora.29.Amd64.Open\)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-29-helix-a12566d-20191210224553
```

Comment:
Do we need to update these? Or can we just remove them as fullMatrix is now always true?

Reply:
I did that at first; but maintaining this list seems easier than putting the whole thing back if we decide that always doing fullMatrix doesn't go so well.

Reply:
Based on the low number of PRs in release/3.1 this should be fine. I can't think of any reason why we would need to switch back.